apps/codeforafrica/src/lib/api.ghost/posts.js

```diff
@@ -0,0 +1,97 @@
+import fs from "fs";
+import path from "path";
+
+import GhostContentAPI from "@tryghost/content-api";
+
+import convertToCamelCase from "@/codeforafrica/utils/camelcaseKeys";
+
+const cacheDir = path.join(process.env.PWD, "public/data");
+
+function ghostAPI() {
+  return new GhostContentAPI({
+    url: process.env.GHOST_URL,
+    key: process.env.GHOST_API_KEY,
+    version: process.env.GHOST_API_VERSION || "v5.0",
+  });
+}
```

Comment:
```suggestion
import { promises as fs } from "fs";
import path from "path";

import camelcaseKeys from "camelcase-keys";
import GhostContentAPI from "@tryghost/content-api";

import equalsIgnoreCase from "@/codeforafrica/utils/equalsIgnoreCase";

const cacheDir = path.join(process.env.PWD, "public/data");

function initializeContentAPI() {
  return new GhostContentAPI({
    url: process.env.GHOST_URL,
    key: process.env.GHOST_API_KEY,
    version: process.env.GHOST_API_VERSION || "v5.0",
  });
}
```

1. since we're doing async for fetch, lets do async for reading files as well.
2. We're already in api.ghost so we shouldn't repeat Ghost if we can.